docusaurus.config.js

```diff
@@ -225,6 +225,14 @@ const config = {
         apiKey: "d7514a8ad9b1aebb64c54172856ecb44",
         indexName: "botkube",
       },
+      announcementBar: {
+        id: "try_botkube",
+        content:
+          '<strong>New to Botkube?</strong> Get started fast with the <a target="_blank" rel="noopener noreferrer" href="https://app.botkube.io">Botkube Cloud Web App</a>.',
```

Comment:
Just a suggestion: WDYT about mentioning that it's free? And also you can consider using an emoji to make it a bit more attractive, like 🤖?

Reply:
Added the free comment. Tried emojis and I don't like how it looks, it distracts from the text.